spec/system/logins_spec.rb

```diff
@@ -29,6 +29,30 @@
       expect(user.role).to eq("challenge_manager")
       expect(page).to(be_axe_clean)
     end
+
+    it "manage submissions index page is accessible with no challenges" do
+      visit manage_submissions_path
+      expect(user.role).to eq("challenge_manager")
+      expect(page).to(be_axe_clean)
+    end
+
+    it "manage submissions index page is accessible with one challenge" do
+      challenge = create_challenge(user: user, title: "Boston Tea Party Cleanup")
+      phase = create_phase(challenge_id: challenge.id)
+      visit manage_submissions_path
+      expect(user.role).to eq("challenge_manager")
+      expect(page).to have_content("Boston Tea Party Cleanup")
+      expect(page).to(be_axe_clean)
+    end
+
+    it "manage submissions my challenge phase page is accessible with one challenge" do
```

Comment:
can we add a spec that expects a challenge manager not to be able to access a challenge phase they aren't assigned to? (validating the comment above on the controller)